iwyu.cc

```diff
@@ -3141,8 +3142,9 @@ class IwyuBaseAstVisitor : public BaseAstVisitor<Derived> {
   bool VisitNestedNameSpecifier(NestedNameSpecifier* nns) {
     // If this is a NamespaceAlias, then mark the alias as used,
     // requiring whichever file(s) declare the alias.
-    if (nns->getKind() == NestedNameSpecifier::NamespaceAlias) {
-      ReportDeclUse(CurrentLoc(), nns->getAsNamespaceAlias());
+    if (auto* alias =
```

Comment:
Could you make this `const auto*`?

Reply:
Yeah, but I find type deduction hides relevant information here. The constness is an interesting property whether we spell out the type or not.